SupportFiles/jira_plugin-install.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
+#
+# Script to download and install JIRA Plugins from other sources
+#
+#################################################################
+# shellcheck disable=SC2086
+PROGNAME="$(basename ${0})"
+SCRIPTHOME="${HOME:-/root}"
+
+JIRADCHOME="/var/atlassian"
+JIRAINSTDIR="/opt/atlassian/jira"
+DBCFGDIR="${JIRADCHOME}/application-data/jira"
+PLUGINS1DIR="${JIRAINSTDIR}/atlassian-jira/WEB-INF/lib"
+PLUGINS2DIR="${DBCFGDIR}/plugins"
+INSTLPLUGINS2DIR="${DBCFGDIR}/plugins/installed-plugins"
+
+# Array of URLs to download plugins (Type 1 or 2)
+PLUGINS1JARURL=(
+                 # <http or https link to jar file>
+               )
+PLUGINS2JARURL=(
+                 # <http or https link to jar or obr file>
+               )
+
+# Install Unzip if needed
```

Comment:
Nit/non-critical:

Generally prefer that programmatic element happen either within a function or within the main program body. Please move this section into the main program or put in a function.

Reply:
Yup. Just a consistency thing. Like I said, "nit/non-critical".

I can't seem to figure out how to comment on a _block_ of content, just individual lines. :-\